.github/workflows/graph-checks.yml

```
@@ -0,0 +1,33 @@
+# This workflow will run graph tests
+
+name: Graph Checks
+
+on:
+  workflow_dispatch:
+  labels:
```

actionlint (1.7.7)

unknown Webhook event "labels". see https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#webhook-events for list of all Webhook event names

(events)